src/main/java/org/sonarsource/scanner/cli/Conf.java

```diff
@@ -71,7 +72,7 @@ Properties properties() {
 
   boolean isSonarCloud(){
     String hostUrl = properties().getProperty(PROPERTY_SONAR_HOST_URL);
-    return hostUrl.toLowerCase().contains("sonarcloud") ? true : false;
+    return hostUrl.toLowerCase(Locale.getDefault()).contains("sonarcloud") ? true : false;
```

Comment:
Is `hostUrl` garanteed to be non null at this point?

Reply:
I suppose that this is not guaranteed, but functionnaly speaking, as long as we are already on the CLi, it should be filled (is there any mechanism that fills this property ? ). But, we never know, so i should defo add a nullcheck here for sure.